mock-core-configs/etc/mock/templates/navy-8.tpl

```diff
@@ -0,0 +1,69 @@
+config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ redhat-rpm-config system-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep'
+config_opts['dist'] = 'el8'  # only useful for --resultdir variable subst
+config_opts['releasever'] = '8'
+config_opts['package_manager'] = 'yum'
+config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
+config_opts['macros']['%dist'] = '.el8'
+config_opts['bootstrap_image'] = 'navylinux:8'
+config_opts['macros']['vendor'] = "The Navy Linux Project."
+config_opts['macros']['packager'] = "packager@navylinux.org"
```

Comment:
The macros vendor and packager are usually set by the build system. Do you really want to set them here?

Reply:
fixed. Thanks for sharing